generator.rb

```diff
@@ -0,0 +1,93 @@
+# This is the is_vowel? method. It gets used in the run_generator method.
+def is_vowel?(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    return true
+  else
+    return false
+  end
+end
+
+# This is the portmanteau generator method.
+# The inputs are collected inside of the method (no parameters passed).
+def run_generator
+  puts "Please enter the first word: "
+  wordA = gets.chomp
+
+  until wordA.length >= 2
+    puts "Oops! This word needs to consist of at least two characters. 
+    Please enter the first word: "
+    wordA = gets.chomp
+  end
+
+  puts "Thanks! Now enter the second word: "
+  wordB = gets.chomp.downcase
+
+  until wordB.length >= 2
+    puts "Oops! This word needs to consist of at least two characters. 
+    Please enter the second word: "
+    wordB = gets.chomp
+  end
```

Comment:
This seems like it's duplicated code. Can you think of a way to refactor it so it's written once? :)